src/NServiceBus.Metrics/AllMetrics.cs

```diff
@@ -0,0 +1,67 @@
+namespace NServiceBus.Metrics
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using global::Metrics;
+    using global::Metrics.Core;
+    using global::Metrics.Json;
+    using global::Metrics.MetricData;
+    using global::Metrics.Utils;
+
+    /// <summary>
+    /// Allows to define all avaible metrics
+    /// </summary>
+    public static class AllMetrics
+    {
+        internal static IMetricBuilder[] Create() => new IMetricBuilder[]
```

Comment:
I agree but it was so much black magic that relied on reflection that I felt it was not worth it. I could write a test that fails if you forget to enter it there :D, thoughts?